packages/admin/cms-admin/src/userPermissions/UserGrid.tsx

```diff
@@ -52,13 +60,45 @@ export const UserPermissionsUserGrid = ({ toolbarAction, rowAction, actionsColum
             headerName: "",
             sortable: false,
             filterable: false,
-            type: "actions",
+
             align: "right",
             pinned: "right",
-            width: actionsColumnWidth,
             disableExport: true,
             renderCell: (params) => {
-                return <> {rowAction && rowAction(params)}</>;
+                const isCurrentUser = params.row.id === currentUser.id;
+                return (
+                    <>
+                        <Tooltip
+                            title={
+                                isCurrentUser ? (
+                                    <FormattedMessage id="comet.userPermissions.cannotImpersonate" defaultMessage="You can't impersonate yourself" />
```

Comment:
This would be less generic in case another reason impersonation is not possible will be added in the future. 

```suggestion
                                    <FormattedMessage id="comet.userPermissions.cannotImpersonateYourself" defaultMessage="You can't impersonate yourself" />
```

Reply:
428dd81c587840067fb7acf84445c578ce2875a0